Source/Engine/Graphics/Mesh.cs

```diff
@@ -633,5 +633,14 @@ public ushort[] DownloadIndexBufferUShort(bool forceGpu = false)
                 throw new Exception("Failed to download mesh data.");
             return result;
         }
+
+        /// <summary>
+        /// Gets the collision proxy points for the mesh.
+        /// </summary>
+        /// <returns>The triangle points in the collision proxy.</returns>
+        public Float3[] GetCollisionProxyPoints()
```

Comment:
Can you make it `internal` for now? On 1.10 branch there was refactor for mesh data and maybe I'll followup on this code. But for editor-only purpose it's fine.

Reply:
Done